src/reason_pprint_ast.ml

```diff
@@ -2152,7 +2152,7 @@ class printer  ()= object(self:'self)
         ]
       (* EQUAL private_flag BAR constructor_declarations {(Ptype_variant _, $2, None)} *)
       | (Ptype_variant lst, scope, None) ->  [
-          privatize scope [makeList ~break:IfNeed ~postSpace:true ~inline:(true, true) (List.map self#type_variant_leaf lst)]
+          privatize scope [makeList ~break:Always_rec ~postSpace:true ~inline:(true, true) (List.map self#type_variant_leaf lst)]
```

Comment:
This seems right.

See also: https://github.com/mjambon/easy-format/blob/6d16b6a9c71fa98386ba785de159dc14bc385ad0/easy_format.mli#L48